locales-src/extra_aliases.js

```diff
@@ -11,204 +11,224 @@
-    अंत: "end",
+    "%1 डिग्री से बाएं घूम जाए": "MOTION_TURNLEFT",
+    "%1 डिग्री से दाएं घूम जाए": "MOTION_TURNRIGHT",
+    "जब झंडे को क्लिक किया गया हो": "EVENT_WHENFLAGCLICKED",
+    अंत: "scratchblocks:end",
   },
 
   ro: {
     // Romanian
-    "rotește la stânga %1 grade": "turn @turnLeft %1 degrees",
-    "rotește la dreapta %1 grade": "turn @turnRight %1 degrees",
-    "când se face click pe stegulețul verde": "when @greenFlag clicked",
-    terminare: "end",
+    "rotește la stânga %1 grade": "MOTION_TURNLEFT",
+    "rotește la dreapta %1 grade": "MOTION_TURNRIGHT",
+    "când se face click pe stegulețul verde": "EVENT_WHENFLAGCLICKED",
+    terminare: "scratchblocks:end",
   },
 
   id: {
     // Indonesian
-    "putar ke kiri %1 derajat": "turn @turnLeft %1 degrees",
-    "putar ke kanan %1 derajat": "turn @turnRight %1 degrees",
-    "ketika bendera hijau diklik": "when @greenFlag clicked",
-    selesai: "end",
+    "putar ke kiri %1 derajat": "MOTION_TURNLEFT",
+    "putar ke kanan %1 derajat": "MOTION_TURNRIGHT",
+    "ketika bendera hijau diklik": "EVENT_WHENFLAGCLICKED",
+    selesai: "scratchblocks:end",
   },
 
   hr: {
     // Croatian
-    "skreni lijevo %1 stupnjeva": "turn @turnLeft %1 degrees",
-    "skreni desno %1 stupnjeva": "turn @turnRight %1 degrees",
-    "kada je zelena zastava kliknut": "when @greenFlag clicked",
-    kraj: "end",
+    "skreni lijevo %1 stupnjeva": "MOTION_TURNLEFT",
+    "skreni desno %1 stupnjeva": "MOTION_TURNRIGHT",
+    "kada je zelena zastava kliknut": "EVENT_WHENFLAGCLICKED",
+    kraj: "scratchblocks:end",
   },
 
   cs: {
     // Czech
-    "otoč se o %1 stupňů doleva": "turn @turnLeft %1 degrees",
-    "otoč se o %1 stupňů doprava": "turn @turnRight %1 degrees",
-    "po kliknutí na zelenou vlajku": "when @greenFlag clicked",
-    konec: "end",
+    "otoč se o %1 stupňů doleva": "MOTION_TURNLEFT",
+    "otoč se o %1 stupňů doprava": "MOTION_TURNRIGHT",
+    "po kliknutí na zelenou vlajku": "EVENT_WHENFLAGCLICKED",
+    konec: "scratchblocks:end",
   },
 
   sl: {
     // Slovenian
-    "obrni se za %1 stopinj v levo": "turn @turnLeft %1 degrees",
-    "obrni se za %1 stopinj v desno": "turn @turnRight %1 degrees",
-    "ko je kliknjena zelena zastavica": "when @greenFlag clicked",
-    ustavi: "end",
+    "obrni se za %1 stopinj v levo": "MOTION_TURNLEFT",
+    "obrni se za %1 stopinj v desno": "MOTION_TURNRIGHT",
+    "ko je kliknjena zelena zastavica": "EVENT_WHENFLAGCLICKED",
+    ustavi: "scratchblocks:end",
   },
 
   fa: {
     // Persian
-    "به اندازه %1 درجه به چپ بچرخ": "turn @turnLeft %1 degrees",
-    "به اندازه %1 درجه به راست بچرخ": "turn @turnRight %1 degrees",
-    "وقتی که پرچم کلیک شد": "when @greenFlag clicked",
-    آخر: "end",
+    "به اندازه %1 درجه به چپ بچرخ": "MOTION_TURNLEFT",
+    "به اندازه %1 درجه به راست بچرخ": "MOTION_TURNRIGHT",
+    "وقتی که پرچم کلیک شد": "EVENT_WHENFLAGCLICKED",
+    آخر: "scratchblocks:end",
+  },
+
+  ja: {
```

Comment:
Who translated these? Do you speak Japanese? 😮

Reply:
they are native to japan

Reply:
Oh nice, I had no idea!

Reply:
Yep; I am actually a Japanese translator for Scratch (for 3 years or so)